students/esokada/lesson03/mailroom.py

```diff
@@ -0,0 +1,75 @@
+#imports
+
+import sys
+
+from statistics import mean
+
+from operator import itemgetter
+
+# data structure
+
+donors = [["Usagi", [12.5, 11.17, 130.1]], ["Ami", [17, 230, 2115]],
+["Rei", [510.1, 50]], ["Makoto", [324, 22.7]], ["Minako", [310]]]
+
+
+#functions
+
+def send_thankyou():
+    donorname = input("Please input a name:\n")
+    if donorname == "cancel":
+        return
+    if donorname == "list":
+        for i in donors:
+            print(i[0])
+        donorname = input("Please input a name:\n")
+        if donorname == "cancel":
+            return
+    donationamount = input("Please enter a donation amount:\n")
+    if donationamount == "cancel":
+        return
+    existingdonor = False
+    for d in donors:
+        if d[0] == donorname:
+            existingdonor = True
```

Comment:
use `is` with True/False

`==` is used for primitive objects like ints/strings

`is` is used for object comparison (like class objects, True/False, None)